.codeclimate.yml

```diff
@@ -0,0 +1,8 @@
+---
+engines:
+  pep8:
+    enabled: true
+ratings:
+  paths:
+  - "**.py"
```

Comment:
We don't need `ratings: paths:` now in quality model world. I think this repo is still on Platform though, so I'll go ahead and migrate it. 

Also, wdyt about omitting exclude_paths if there are none